lib/main-production.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
```

Comment:
I think the file name should follow snake_case convention, how do you think? 🙏

Reply:
It's generated by flutter_flavorizr, so we can change it but have to this again if we re-generate them 😢

Reply:
oh 🧱

Reply:
we can think about it later when we create our own template. Under the hood flavorizr just runs some scripts to create boilerplate setup file for us, we can do this ourselves and remove flavorizr later and follow our correct convention. The current setup helps us move faster with the early learning step.